src/commands/test.js

```diff
@@ -11,6 +14,10 @@ const test = (context) => {
     extraEnv.DETAILED_LOG_TO_STDOUT = 'true';
   }
 
+  if (process.version !== LAMBDA_VERSION) {
+    throw new Error(`You're running tests on Node ${process.version}, but Zapier runs your code on ${LAMBDA_VERSION}. To ensure errors don't sneak up on you, the version numbers must match.`);
```

Comment:
Since we include a `.nvmrc` I'd add something like "Install and run `nvm use` to switch (https://github.com/creationix/nvm)."